crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -970,10 +975,11 @@ impl RoomInfo {
     }
 
     fn creator(&self) -> Option<&UserId> {
-        Some(match self.base_info.create.as_ref()? {
-            MinimalStateEvent::Original(ev) => &ev.content.creator,
-            MinimalStateEvent::Redacted(ev) => &ev.content.creator,
-        })
+        #[allow(deprecated)]
```

Comment:
This is a temporary solution far from perfect. The idea is that with room version 11, the `creator` field is gone and the `sender` field should be used instead. The upcoming Matrix v1.8 spec says that clients should just always use the `sender` of the event, which we don't have here.

I see 3 options that are too intrusive for this PR:
1. Do not use the Ruma type, but create a new one where the creator is still mandatory and copied from the sender.
2. Change `MinimalStateEvent` to include `sender` and do not store these events in the room info in the state store, but load them from the room state events. An advantage of this is that we can always add new fields/data without requiring database migrations.
3. Change `MinimalStateEvent` to include `sender` and make database migrations to fix the data in the store.